### PR TITLE
Update to MCAS section of the Platform design

### PR DESCRIPTION
**Pull request by SamBH**
Update to MCAS section in the Platform design to reflect changes to the product and layout in the console. This includes technical and readability updates to align the design more closely with the configuration as presented in the console.

Note, Microsoft has announced that MCAC will be changing names to Defender for Cloud Apps but this will not be reflected in the Blueprint until Microsoft has updated their documentation online.

### CODE REVIEW

File: blueprint/platform.md [hunk 1]
```diff
@@ -565,9 +565,9 @@ Microsoft Defender for Endpoint integration | Enabled | To provide additional vi
 Enforce app access | Enabled | To block access to apps marked as Unsanctioned.
 List of sanctioned and unsanctioned cloud apps | To be developed during build with the Agency's Cyber Intelligence team. | Provides visibility within the Agency as to what cloud applications are in use and by which department within the Agency.
 
-### MCAS - Log collector
+#### Automatic log upload & Log collectors
```
Comment (zedzedzed) [inline blueprint/platform.md:568]: Off our discussion around log collectors, could you please check the implementation applicability for this section and MCAS - Cloud Discovery are accurate?

Comment (zedzedzed) [reply blueprint/platform.md:568]: That is, are they still for all implementation types?

Comment (SamBH) [reply blueprint/platform.md:568]: Have updated the design and pushed another commit.

File: blueprint/platform.md [hunk 2]
```diff
@@ -571,14 +571,11 @@ A log collector receives logs from supported firewall and proxy devices, providi
...
-There are two supported deployment modes for log collectors:
-
-* Docker container – a Microsoft-provided Docker image for both Windows and Linux operating systems.
-* Virtual appliance – a Microsoft-provided Virtual Machine (VM) image for Hyper-V and VMware hypervisors. Note, the virtual appliance deployment mode is now deprecated.
+Log collectors are deployed as a Docker container using a Microsoft-provided Docker image for both Windows and Linux operating systems.
 
 The Docker container is supported on Windows 10 and Windows Server version 1709 and later, Ubuntu versions 14.04, 16.04 and 18.04, and Red Hat Enterprise Linux (RHEL) and CentOS 7.2 or later. The image can be deployed on VMs either hosted on-premises or within Azure, provided suitable network connectivity from the proxy/firewall devices is available.
 
-Log Collector Design Decisions for all agencies and implementation types.
+Log Collector Design Decisions for agencies that use a centralised internet gateway.
```
Comment (zedzedzed) [inline blueprint/platform.md:578]: Just confirming that this can still apply to both cloud-native and hybrid environments?

Comment (SamBH) [reply blueprint/platform.md:578]: Yes, as long as they have a centralised internet gateway, doesn't matter if it is on-premises (hybrid) or in Azure (cloud deployment).

**Review by zedzedzed — APPROVED** [review]

**Pull request merged**